src/app/shared/utils/object-helper.ts

```diff
@@ -0,0 +1,17 @@
+/* возвращает свойство объекта по передаваемому пути
+ * ```typescript
+ *  const a = {b: { c: 1}};
+ *  getProperty(a, 'b.c') // 1
+ * ```
+ *
+ * @param obj передаваемый объект
+ * @param path путь к свойству через '.'
+ *
+ * @returns значение свойства в объекте
+*/
+export function getPropertyFromPath(obj: any, path: string): any {
+  const pathArr = path.split('.');
+
+  if (pathArr.length > 1) return getPropertyFromPath(obj[pathArr[0]], pathArr.slice(1).join('.'));
```

Comment:
Так if лучше не форматировать: "JavaScript allows the omission of curly braces when a block contains only one statement. However, it is considered by many to be best practice to never omit curly braces around blocks, even when they are optional, because it can lead to bugs and reduces code clarity."
```suggestion
if (pathArr.length > 1) {
    return getPropertyFromPath(obj[pathArr[0]], pathArr.slice(1).join('.'));
  } else {
    return obj[pathArr[0]];
}
```